tests/test_config.py

```diff
@@ -255,3 +255,13 @@ def test_style_invalid_choice(self, capsys: pytest.CaptureFixture[str]) -> None:
         output = capsys.readouterr()
         assert not output.out
         assert "--style: invalid choice: 'invalid'" in output.err
+
+    def test_style_in_toml(self, monkeypatch: pytest.MonkeyPatch) -> None:
```

Comment:
Wonderful!

Should we also test for multiple styles? We support that on the command line so we should probably support that in `toml` as well?

Reply:
done